### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by CaptainMorganCodes**
I updated the images to reflect our new messaging
Working on updating & finalizing readme. Suggestions are welcome! =)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,12 @@
...
-[Travis CI](https://www.travis-ci.com/401n14/midTerm)
+* [Project link](https://n14-transcribe.herokuapp.com)
+
+* [Travis CI](https://www.travis-ci.com/401n14/midTerm)
+
+* [Documentation](https://n14-transcribe.herokuapp.com/docs/)
+
 ---------------------------------
 ## Web Application
 ***[Explain your app, should be at least a paragraph. What does it do? Why should I use? Sell your product!]***
```
Comment (all1124) [inline README.md]: ***[Explain your app, should be at least a paragraph. What does it do? Why should I use? Sell your product!]***
^ Should this be removed?

File: README.md [hunk 2]
```diff
@@ -49,13 +53,19 @@ Start chatting
...
 
 ### Enter a username
-![username](./images/NameLogin.png)
+![username](./images/enterUsername.png)
 
 ### Type in your preferred language
-![Language](./images/LanguageChoice.png)
+![Language](./images/hello.png)
+
+### The last 5 messages from the chat will be displayed
```
Comment (all1124) [inline README.md:61]: Chat history display has been changed. Can you update the image?

File: README.md (hunk 1 above)
Comment (CaptainMorganCodes) [reply README.md]: This is updated

File: README.md (hunk 2 above)
Comment (CaptainMorganCodes) [reply README.md:61]: I updated the image

**Review by NadyaIlinskiy — APPROVED** [review]

**Pull request merged**